rwengine/src/objects/CharacterObject.cpp

```diff
@@ -35,6 +35,7 @@ CharacterObject::CharacterObject(GameWorld* engine, const glm::vec3& pos,
     , m_look(0.f, glm::half_pi<float>())
     , running(false)
     , jumped(false)
+    , jumpAnimation(NULL)
```

Comment:
Instead of NULL you should use nullptr.

Reply:
Ah thanks, it shows that I've used to C over C++...